app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -41,4 +42,8 @@ class AdministratorControlsActivity : InjectableAppCompatActivity(), RouteToAppV
       return KEY_NAVIGATION_PROFILE_ID
     }
   }
+
```

Comment:
The ordering in this is incorrect. Either keep companion on the top or at the bottom of the class

Reply:
Done